src/views/projectExplorer/index.ts

```diff
@@ -844,6 +844,16 @@ export default class ProjectExplorer implements TreeDataProvider<ProjectExplorer
             await commands.executeCommand(`code-for-ibmi.setDeployLocation`, undefined, workspaceFolder, defaultDeployLocation);
           }
         }
+      }),
+      commands.registerCommand(`vscode-ibmi-projectexplorer.connect`, async () => {
+        const connections = workspace.getConfiguration(`code-for-ibmi`).get<ConnectionData[]>('connections');
+				if (connections && connections.length) {
+					await commands.executeCommand(`code-for-ibmi.connectTo`, connections.length === 1 ? connections[0].name : undefined);
```

Comment:
Strange, this is already how the quick pick from `code-for-ibmi.connectTo` is supposed to work:
![Code_SAFIwGz63W](https://github.com/IBM/vscode-ibmi-projectexplorer/assets/11096890/673368f4-87e2-4956-acac-9cc37646b5ff)

How many connections do you have in your browser?!

Reply:
In the server browser, I have 2 connections defined which I can also see when I use this action since I have recently connected to them. But if I create a new connection (but only Save & Exit), then I don't see it in the quick pick.

It looks like instead of showing all connections, it only shows connections which have been connected to at least once?

Reply:
Also just noticed that the top of the quick pick does say `Last IBM i connections` which I guess matches the current behaviour. Maybe this should be changed to show all?